concent_api/command_line_tool/key_manager.py

```diff
@@ -0,0 +1,59 @@
+from django.conf import settings
+from utils.testing_helpers import generate_ecc_key_pair
+
+REQUESTOR_PRIVATE_KEY = b"\xf1)\x82\xfd\xa7t\xd5\x7f\x00K\xa9\x12\x80\xed\x88\xef\xae\x03;\x11<0\xf8b\x96@\xaa\x92\xef\xa6;'"
+REQUESTOR_PUBLIC_KEY = b'\xe3\xb0zQ\xef\xf3\xed\x82K\xe6^@ \x0b^\xd9\xbd\x05q\r\xc5\xda)\x96G\x89\xd6\x9c\x83[v\xa4C\xdf\x1d2\x03\xbfc\xd2\x05^\xa0\xae\xc2\xfb\xd5\xf4\xcf\xb9_\xb0r\xac\x93\xe2\xbd\xed\xaf\xb5l\xa25T'
+
+PROVIDER_PRIVATE_KEY = b'\n\xf3bqtu`\xa6q\x08~\xfd\x98\xeb&8J\xb4\xcb\xe7\xe2\xd4\xd0J\x1fB\xdc|!j@\t'
+PROVIDER_PUBLIC_KEY = b'CYz\xf3\x85\x82\xa0(\x86\xd5\x9f\xcdF\x01\x1d\xf5\x8a\x84\xc7\xf4\xd6Y\xbe\xc703\xdfrK,\n\xe6\x94JG\xcfr\xaf\x82{\xe8\xd68f\xfc\x05l\x13\n\x1a\x1c\x11 \x83p \xa1\xf7"ab\xd0\x1d\xd6'
+
+
+class WrongConfigurationException(Exception):
+    def __init__(self, error_message):
+        self.message = error_message
+
+
+def are_keys_predifined(party):
+    formatted_party = party.upper()
```

Comment:
There should be an assert that `party is in ['provider', 'requestor']`

Reply:
This is handled during parsing arguments.